### ISSUE TITLE
How to add all release files that end with file extension

### ISSUE THREAD

**Issue by chancesmith** [gruntwork-io/fetch#84]
Can you use the release-asset regex to get many files that match a file extension?

`fetch --repo="https://github.com/foo/bar" --tag="0.1.5" --release-asset="*.tar.gz" /tmp`

similar to #58, but for any other files that match

![Screen Shot 2021-01-19 at 3 20 13 PM](https://user-images.githubusercontent.com/4466585/105094450-eb427880-5a69-11eb-8df9-eada18025e75.png)

**Comment by brikis98** [comment]
I don't think this is currently supported, but would be handy to add. PR welcome!

**Comment by vkarhaltsev** [comment]
I was just able to do the described job (fetch version - 0.4.1):
```
./fetch_linux_amd64 --tag v1.0.0 --repo https://github.com/noxdafox/rabbitmq-cloudwatch-exporter --release-asset=".*.ez" /tmp/
```
